prime-router/src/test/kotlin/azure/MessagesFunctionsTests.kt

```diff
@@ -413,6 +419,8 @@ class MessagesFunctionsTests {
         assert(res.status.equals(HttpStatus.OK))
 
         clearAllMocks()
```

Comment:
I'm not exactly sure what the above assertion does as the setup seems to mock the function under test; I left it alone, but because of the `clearAllMocks` call I did need to mock again.